libs/checkout/state/src/selectors/placed-order.selector.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  createSelector,
+  MemoizedSelector,
+} from '@ngrx/store';
+
+import { getDaffCartSelectors } from '@daffodil/cart/state';
+import { DaffOrder } from '@daffodil/order';
+import { getDaffOrderSelectors } from '@daffodil/order/state';
+
+import { DaffCheckoutStateRootSlice } from '../reducers/public_api';
+
+export interface DaffCheckoutPlacedOrderSelectors<T extends DaffOrder = DaffOrder> {
+  /**
+   * Selector for the most recently placed order (if any).
+   */
+  selectPlacedOrder: MemoizedSelector<DaffCheckoutStateRootSlice<T>, T>;
+  /**
+   * Selector for the existence of the most recently placed order.
+   */
+  selectHasPlacedOrder: MemoizedSelector<DaffCheckoutStateRootSlice<T>, boolean>;
+}
+
+const createCheckoutPlacedOrderSelectors = <T extends DaffOrder = DaffOrder>() => {
```

Comment:
This function needs a return type specified.

Reply:
fixed